README.md

```diff
@@ -84,6 +83,36 @@ To run it from your desktop (Windows, Mac, Linux) it can be a bit more involved.
 Once that is done you can open up a command line in the project's directory and run `lime test windows -debug`. Once that command finishes (it takes forever even on a higher end PC), you can run FNF from the .exe file under export\release\windows\bin
 As for Mac, 'lime test mac -debug' should work, if not the internet surely has a guide on how to compile Haxe stuff for Mac.
 
+### FNF now supports [VSCode Devcontainers](https://code.visualstudio.com/docs/remote/containers)
+
+There are some VSCode task created for compiling and running HTML5 versions of FNF inside a devcontainer

```

Comment:
Grammar check: "Provided inside of the repository are some VS Code tasks created for compiling and running HTML5 builds instead of a development container."